includes/admin/class-sensei-extensions.php

```diff
@@ -50,9 +50,26 @@ public function init() {
 	 */
 	public function enqueue_admin_assets() {
 		$screen = get_current_screen();
+
 		if ( in_array( $screen->id, [ 'sensei-lms_page_sensei-extensions' ], true ) ) {
 			Sensei()->assets->enqueue( 'sensei-extensions', 'extensions/index.js', [], true );
 			Sensei()->assets->enqueue( 'sensei-extensions-style', 'extensions/extensions.css' );
+
+			wp_localize_script(
+				'sensei-extensions',
+				'sensei_extensions',
+				[
+					'connectUrl' => add_query_arg(
```

Comment:
Maybe after rebasing it, we could add the `connectUrl` to the same extensions request?